canonical/canonical.com
# Update partnerlogos

## Done

- Updated partner logo template as per ticket

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Visit all partners pages and make sure everything looks as expected 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5075
